client/app/account/contacts/service/contacts-service.service.js

```diff
@@ -0,0 +1,49 @@
+import _ from 'lodash';
+import BillingService from '../../../models/BillingService.class';
+
+import { AVAILABLE_SERVICES } from './user-contacts.constants';
+
+export default class {
+  /* @ngInject */
+  constructor(OvhApiOvhProduct, OvhHttp) {
+    this.OvhApiOvhProduct = OvhApiOvhProduct;
+    this.OvhHttp = OvhHttp;
+  }
+
+  // TODO: Find a way to inject ovh-api-services depending on the service category
```

Comment:
Looks like a good requirement for the successor of `ovh-api-services`, maybe you could add it to the document of requirements (ask JC for it as I don't know where it is :D)